src/atoms/heading/Heading.scss

```diff
@@ -1,17 +1,22 @@
 @import '../../../assets/styles/_globals.scss';
 
-$headings-margin                : 0 0 0.5em 0 !default;
-$headings-font-family           : $font-family-base !default;
-$headings-font-weight           : 700 !default;
-$headings-line-height           : $font-line-height !default;
-$headings-color                 : $color-secondary !default;
-$headings-text-transform        : none !default;
-$heading-font-size--first-level : 18px !default;
-$heading-font-size--second-level: 18px !default;
-$heading-font-size--third-level : 14px !default;
-$heading-font-size--fourth-level: 14px !default;
-$heading-font-size--fifth-level : 12px !default;
-$heading-font-size--sixth-level : 12px !default;
+$headings-margin                          : 0 0 0.5em 0 !default;
+$headings-font-family                     : $font-family-secondary !default;
+$headings-font-family--secondary          : $font-family-base !default;
+$headings-font-weight                     : $font-weight-medium !default;
+$headings-line-height                     : $font-line-height !default;
+$headings-color                           : $color-primary !default;
+$headings-text-transform                  : uppercase !default;
+
+$heading-font-size--first-level\@screen-m : $font-size-super-extra-large !default;
+$heading-font-size--first-level           : $font-size-extra-large !default;
+$heading-font-size--second-level\@screen-m: $font-size-extra-large !default;
+$heading-font-size--second-level          : $font-size-large !default;
+$heading-font-size--third-level\@screen-m : $font-size-large !default;
```

Comment:
IMO we should do it as simple as possible. Maybe it would be good do keep 1 size for one heading for now? @anqaka WDYT?

Reply:
Regarding to UX, I think smaller font-size for h1 and h2 for mobile view is often requested and useful, not sure about h3

Reply:
ok, lets leave this problem for now